app/kumactl/cmd/install/install_transparent_proxy_test.go

```diff
@@ -23,7 +23,8 @@ var _ = Describe("kumactl install transparent proxy", func() {
 	})
 
 	type testCase struct {
-		extraArgs    []string
+		skip      func(stdout, stderr *bytes.Buffer) bool
+		extraArgs []string
```

Comment:
nitpick: I think this will fail linter (if not, ignore)

```suggestion
		skip         func(stdout, stderr *bytes.Buffer) bool
		extraArgs    []string
```